dcos/package.py

```diff
@@ -1991,6 +1941,55 @@ def get_package_version(self, package_name, package_version):
                               pkg.registry(), pkg.get_path())
 
 
+    def installed_apps(package_name, app_id, endpoints=False):
```

Comment:
We are ignoring package_name and app_id. This is being used by the caller. We need to fix this but it is okay for now.